inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -134,18 +134,16 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
     --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-main-bg: rgb(247, 247, 247);
     --#{$prefix}card-border-color: var(--bslib-dashboard-border-color-translucent, var(--#{$prefix}border-color-translucent));
-    --bslib-dashboard-card-cap-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-card-header-font-weight: #{$font-weight-semibold};
-    --bslib-sidebar-bg: var(--#{$prefix}body-bg);
-    --bslib-sidebar-fg: var(--#{$prefix}color);
+    --bslib-sidebar-bg: RGBA(var(--#{$prefix}body-bg-rgb), 0.1);
   }
 
   @include color-mode(dark) {
     --bslib-dashboard-main-bg: rgb(20, 20, 24);
   }
 
   .bslib-card {
-    --#{$prefix}card-cap-bg: var(--bslib-dashboard-card-cap-bg);
+    --#{$prefix}card-cap-bg: ;
```

Comment:
Bootstrap uses `$card-cap-bg: rgba(var(--#{$prefix}body-color-rgb), .03) !default;` and we don't want to get in the way of that -- we still want that value if dashboard design is disabled. This takes the `background-color: var(--bs-card-cap-bg)` out of play, letting the card cap inherit as needed, but also letting users still set `--bs-card-cap-bg` if they want.

Reply:
This is very useful context that seems useful for development -- let's put it in a comment?